profiles/common/themes/ec_resp/css/interinstitutional_less/hack-header.less

```diff
@@ -157,6 +159,29 @@
   color:#000;
 }
 
+.lang-select-site {
+  position: absolute;
+  z-index: 1024;
+  right:15px;
+  border-style:none;
+  box-shadow:none;  
+  height:23px;
+  .panel-body {
+    padding:0px;
+    .lang-select-site__link {
+      display: block;
+      margin: -15px;
+      padding: 15px;
+    }
+    .lang-select-site__code {
+      .label;
+      .label-primary;
+      margin-right: 4px;
+    }
+  }
+}
+
+
 /* ------------------ Hack when js is enabled ------------------ */
```

Comment:
Coding standards: there should only be a single blank line to separate a ruleset and a comment.